server/src/main/java/com/paypal/selion/utils/SauceLabsRestApi.java

```diff
@@ -39,6 +43,48 @@
     private final int sauceTimeout;
     private final int sauceRetryCount;
     private int maxTestCase = -1;
+    private Map<String, Boolean> accountCache;
+
+    final class SauceLabsHttpResponse {
+        private StringBuilder entity;
+        private int status;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'status' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2418638937/issues/source?bid=2928425&fileBranchId=3233060#l50)